site/resources/views/folders/show.blade.php

```diff
@@ -3,22 +3,138 @@
 @section('title')
     {{ $folder->title }}
```

Comment:
Add folder icon before title when the user is in the folder index view ?

![CleanShot 2023-10-30 at 16 24 42](https://github.com/unil-lettres/impact/assets/1675064/ecf24974-3c82-4e3a-8311-3447fd11e960)

Reply:
done